app/presenters/finder_content_item_presenter.rb

```diff
@@ -50,6 +50,10 @@ def details
       summary: metadata.fetch("summary", nil),
       facets: schema.fetch("facets", []),
     }
+    unless metadata["default_order"].nil?
+      details_hash[:default_order] = metadata["default_order"]
```

Comment:
That takes the line to 89 characters, not our suggested 80. Does that matter in this case?